components/organisms/o-shipping.vue

```diff
@@ -0,0 +1,320 @@
+      :title="`2. ${$t('Shipping')}`"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <div
+        v-if="currentUser && hasShippingDetails()"
+        class="form__element form__group"
+      >
+        <SfCheckbox
+          v-model="shipToMyAddress"
+          class="form__checkbox"
+          name="shipToMyAddress"
+          :label="$t('Ship to my default address')"
+        />
+      </div>
+      <SfInput
+        v-model.trim="shipping.firstName"
+        class="form__element form__element--half"
+        name="first-name"
+        :label="$t('First name *')"
+        :valid="!$v.shipping.firstName.$error"
+        :error-message="
+          !$v.shipping.firstName.required
+            ? $t('Field is required')
+            : !$v.shipping.firstName.minLength
+              ? $t('Name must have at least 2 letters.')
+              : ''
+        "
+        @blur="$v.shipping.firstName.$touch()"
+      />
+      <SfInput
+        v-model.trim="shipping.lastName"
+        class="form__element form__element--half form__element--half-even"
+        name="last-name"
+        :label="$t('Last name *')"
+        :valid="!$v.shipping.lastName.$error"
+        :error-message="
+          !$v.shipping.lastName.required ? $t('Field is required') : ''
+        "
+        @blur="$v.shipping.lastName.$touch()"
+      />
+      <SfInput
+        v-model.trim="shipping.streetAddress"
+        class="form__element"
+        name="street-address"
+        :label="$t('Street name *')"
+        :valid="!$v.shipping.streetAddress.$error"
+        :error-message="
+          !$v.shipping.streetAddress.required ? $t('Field is required') : ''
+        "
+        @blur="$v.shipping.streetAddress.$touch()"
+      />
+      <SfInput
+        v-model.trim="shipping.apartmentNumber"
+        class="form__element"
+        name="apartment-number"
+        :label="$t('House/Apartment number *')"
+        :valid="!$v.shipping.apartmentNumber.$error"
+        :error-message="
+          !$v.shipping.apartmentNumber.required ? $t('Field is required') : ''
+        "
+        @blur="$v.shipping.apartmentNumber.$touch()"
+      />
+      <SfInput
+        v-model.trim="shipping.city"
+        class="form__element form__element--half"
+        name="city"
+        :label="$t('City *')"
+        :valid="!$v.shipping.city.$error"
+        :error-message="
+          !$v.shipping.city.required ? $t('Field is required') : ''
+        "
+        @blur="$v.shipping.city.$touch()"
+      />
+      <SfInput
+        v-model.trim="shipping.state"
+        class="form__element form__element--half form__element--half-even"
+        name="state"
+        :label="$t('State / Province')"
+      />
+      <SfInput
+        v-model.trim="shipping.zipCode"
+        class="form__element form__element--half"
+        name="zipCode"
+        :label="$t('Zip-code *')"
+        :valid="!$v.shipping.zipCode.$error"
+        :error-message="
```

Comment:
We don't need to check second condition and this can be typed simpler: `$v.fieldName.requred ? $t('Field is required') : $t('Other message')`

Reply:
Done.